### ISSUE TITLE
rustfmt crashes on unstable/experimental syntax

### ISSUE THREAD

**Issue by umanwizard** [rust-lang/rustfmt#4190]
input:
```
fn f(i: isize) {
    match i {
        1.. => {}, // positive
        ..0 => {}, // negative
        0   => {}, // zero
    }
}
```

output:
```
$ rustfmt test.rs
thread 'main' panicked at 'not implemented', src/tools/rustfmt/src/patterns.rs:203:27
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by calebcartwright** [comment]
What version of rustfmt are you using @umanwizard (`rustfmt --version`)

**Comment by umanwizard** [comment]
`rustfmt 1.4.12-stable (a828ffe 2020-03-11)`

**Comment by calebcartwright** [comment]
Could you try updating to the latest version on stable (1.4.14)

**Comment by umanwizard** [comment]
How? I've done `rustup update` and I still have 1.4.12

**Comment by calebcartwright** [comment]
I thought that 1.4.13 at least had landed on stable already, but perhaps not.

**Comment by calebcartwright** [comment]
This is fixed in 1.4.13 and master so I'm going to go ahead and close this as there's no additional action we can take. 

If using nightly as an option for you then you'll of course have access to newer versions (1.4.15 being the latest). If not then you could consider using `#[rustfmt::skip]` (https://github.com/rust-lang/rustfmt#tips) temporarily til 1.4.13 hits stable.

**Comment by umanwizard** [comment]
@calebcartwright Thank you for the support.
